include/common/configure.h

```diff
@@ -258,6 +259,136 @@ class Configure {
     return StatisticsConf;
   }
 
+  inline auto logCheckError(ErrCode Code, ASTNodeAttr Node) const noexcept {
```

Comment:
Proposed not to use `auto` as return type.

Reply:
Using `auto` as the return type seems to be employed in log functions of `serilaize.h` and `loader.h`. Do I need to modify it?